strings/french_replacement.json

```
@@ -2,40 +2,44 @@
 
     "french": {
 		"shi": "shee",
-		"thi": "thee",
-		" yes ": " oui ",
-		"no": "non",
-       		 "I'm ": "j' ", 
-   	     	" am ": " suis ", 
-		" a ": " un ",
+		"thi": "zee",
+		"them ": "zem ",
+		"that ": "zat ",
+		"will": "weell",
+		"through": "srough",
```

```suggestion
		"through": "zrough",
```
For english speakers, this rolls off the tongue more like the intended accent

---

I will keep this one as is to distinguish the difference between unvoiced and voiced th sounds

---

Yes but there's a difference between ze accent being _"zroo"_ and being _"ssroff"_, hence the z sound making you understand it's under the "th" sphere and not a mixed "s'rough" as in _"it's rough"_

---

if you think it's important for people to be able to understand it I'll add it